### PR TITLE
add ComboBoxInput interface

### PR DESCRIPTION
**Pull request by mikhailUshakoff**

### CODE REVIEW

File: ComboBoxInput/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+`get` - allows to get the item selected by the user.
+
+arguments:
+
+    answerId: uint32 - function id of result callback
+
+    prompt: bytes - utf-8 string to print to the user before input
+
+    items: bytes[] - list of items for selection. (should not be reshuffled by a browser)
```
Comment (BigRoma) [inline ComboBoxInput/README.md:27]: I think we need a structure, in this case, for example, we can select a country and get value like "RU", but normally our need to show the name of the country, like "Russia". 

```
struct ComboBoxItem {
    string label;
    string value;
    bool disabled;
}
```

File: ComboBoxInput/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+returns:
+
+    index: uint256 - index of selected item
```
Comment (BigRoma) [inline ComboBoxInput/README.md:31]: Also, we can return more variants than one.

Comment (BigRoma) [reply ComboBoxInput/README.md:31]: If we will use this case, our need the `bool multiple` property in inputs section

**Review by BigRoma — CHANGES_REQUESTED** [review]
Request changes

**Comment by ilyar** [comment]
@mikhailUshakoff up

**Pull request closed without merging**